flytekit/clis/auth/auth.py

```diff
@@ -0,0 +1,229 @@
+import base64 as _base64
+import hashlib as _hashlib
+import os as _os
+import re as _re
+import requests as _requests
+import webbrowser as _webbrowser
+
+from multiprocessing import Process as _Process, Queue as _Queue
+
+try:  # Python 3.5+
+    from http import HTTPStatus as _StatusCodes
+except ImportError:
+    try:  # Python 3
+        from http import client as _StatusCodes
+    except ImportError:  # Python 2
+        import httplib as _StatusCodes
+from BaseHTTPServer import HTTPServer as _HTTPServer, BaseHTTPRequestHandler as _BaseHTTPRequestHandler
```

Comment:
nit: let's import the module. it makes refactoring easier and keeps the namespace cleaner.

```
import BaseHTTPServer as _base_http_server

Later...
_base_http_server.HTTPServer
```

Reply:
same for urlparse below

Reply:
but it makes it harder to jump into the definition of the underlying object.

Reply:
hm actually i can still jump around in my ide to the underlying object with refactoring